PyInstaller/building/datastruct.py

```diff
@@ -173,6 +178,17 @@ def __postinit__(self):
             else:
                 # create a dict for easier access
                 data = dict(zip((g[0] for g in self._GUTS), data))
+
+                # this ::HACK:: converts winmanifest from a string
+                if is_win and data.get('manifest'):
+                    if not data['manifest'].startswith(winmanifest_marker):
```

Comment:
Marker was put in as a paranoid response to not knowing what else might be present in this field.  I would gladly remove it if someone who knows more about this code  were to say that it isn't necessary. :-)